static/conformance-view.js

```diff
@@ -252,11 +253,21 @@ Conformance.prototype.handleToolbarUI = function () {
 
 Conformance.prototype.handleStatusIcon = function (element, status) {
     if (!element) return;
-    element.attr("class", "status glyphicon glyphicon-" + (status.code === 3 ? "remove-sign" : (status.code === 2 ? "info-sign" : "ok-sign")))
+
+    function glyphClass(code) {
+        return code === 3 ? "remove-sign" : (code === 2 ? "info-sign" : "ok-sign");
```

Comment:
ternaries can be a pain, especailly cascading ones. seing as you've made a nice function maybe:

```
if (code === 3) return "remove-sign";
if (code === 2) return ...;
return ...
```

instead?

Reply:
Nice catch. I just c&p'ed, but this seems a better approach